### DISCUSSION TITLE
Is there a way to turn detection/recording on/off via Home Assistant?

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#14562] [Ask A Question] [answered]
As an example.  I have a camera set on my backyard that detects people.  On the backyard I also have a projector that I use to play movies and sporting events.  When ever I do so the backyard camera creates many events because it detects the people from the projector.  HomeAssistant can tell when the projector is on because it powers up the chrome cast attached to it.  I would like to create an automation that when the projector turns on (chrome cast is "available") it would turn off all detections/recordings from the backyard camera, but when projector turns off, have the backyard camera go back to regular person detection mode.

 Is this possible?

**Comment by NickM-27** [comment]
yes, use the integration https://github.com/blakeblackshear/frigate-hass-integration

**Reply by CancunManny** [reply]
Thank you for quick response.  I already use the frigate hass integration, but not sure what "call action" I would do.  If I search for "camera" options, it allows me to turn on/off motion detection, but I assume frigate continues recording as usual.  When I search for "frigate" actions I only see 3 options, frigate export, frigate favorite and frigate ptz

**Reply by NickM-27** [reply, marked as answer]
Look at the switches
![Screen Shot 2024-10-24 at 14 03 34 PM](https://github.com/user-attachments/assets/85616641-5a85-4103-b1d5-644d86e0ec45)

**Reply by CancunManny** [reply]
Wow, that was easy!  In fact so easy I didn't think it was the solution.  I had assumed it would just turn the sensor off in HA but that something else had to be done to change Frigate itself, but that is not the case.

Just to add a bit more info for someone that might be trying to do something similar.  You can't turn "Motion" off and leave "Detect" on.  If you want to turn both off (as in my case), you FIRST have to turn off Detect, and then you turn Motion off.  If you try to turn off Motion off while Detect is on, Motion will turn on by itself right away.

So to set up the automation you would call acting "switch" to turn on or off.  In my case my camera name is Terraza, but you can substitute Terraza with the actual name of your camera.

![image](https://github.com/user-attachments/assets/d89ba35e-7c23-487f-939f-5b25195cf544)
